.replit

```diff
@@ -1,2 +1,10 @@
+run = "npm start"
+entrypoint = "server.js"
+
+[packager]
 language = "nodejs"
-run = "npm start"
\ No newline at end of file
+
+  [packager.features]
+  packageSearch = true
+  guessImports = true
```

Comment:
There's a clear drawback of this setting in this challenge. Because there's `moongose` is required in `server.js`, when run, it is added automatically to the `package.json`. As that's part of the task, that'd be a bit weird.

However with the `guessImports = false`, the install on _Run_ doesn't happen. So the installs in the other boilerplates also must have been _just_ because of the guessing and it'd be better to turn that off.

What do you think about changing this setting to `false` and changing run command to `npm install; npm start` as @RandellDawson suggested https://github.com/freeCodeCamp/freeCodeCamp/issues/46824#issuecomment-1180669463? In other boilerplates as well.

Reply:
Your right, nice catch. I guess that's just how they deal with dependencies (both the docs and the official Replit Node starter). I really do not understand why they change it so the packages are not installed before the npm run command.

I guess I will just update the PRs. I think we should keep them the same so even if it isn't an issue for some of them we might as well use the same configs.

Reply:
@gikf One thing I did notice. Now it will run install every time you run the Replit (which makes sense). I assume that also applies to projects using nodemon and when you use the save hotkeys which they have set up to re-run the Replit.

I'm also not sure if it is starting the server correctly. It logs out that it started but it doesn't seem to start the preview as expected when I tested it.

I think we have to look at this some more before making the changes.

Reply:
Some more eyes definitely wouldn't hurt. For what it's worth, I'm getting the preview window displayed
![image](https://user-images.githubusercontent.com/60067306/178571349-4cd98066-e541-41a4-9889-d56421dfdb66.png)

Reply:
Hmm, I did remove the other config options but I wouldn't have thought that mattered. Maybe it was just a fluke.

---

Having it install the dependencies every time you stop and start definitely seems a bit annoying to me.